static/app/components/core/alert.chonk.tsx

```diff
@@ -0,0 +1,126 @@
+import {css, type DO_NOT_USE_ChonkTheme} from '@emotion/react';
+
+import type {AlertProps} from 'sentry/components/alert';
+import type {ChonkPropMapping} from 'sentry/utils/theme/withChonk';
+
+const chonkAlertPropMapping: ChonkPropMapping<AlertProps, ChonkAlertProps> = props => {
+  return {
+    ...props,
+    type:
+      props.type === 'muted' ? 'subtle' : props.type === 'error' ? 'danger' : props.type,
+  };
+};
+
+interface ChonkAlertProps extends Omit<AlertProps, 'type'> {
+  theme: DO_NOT_USE_ChonkTheme;
+  type: 'subtle' | 'info' | 'warning' | 'success' | 'danger';
+  size?: 'sm';
+}
```

Comment:
We'll discuss this during our chat, but this is one way in which we can swap components and rely on the new theme values without touching all the stateful parts